e2e/steps/test_map_creation.py

```diff
@@ -21,58 +21,55 @@ def logged_in_and_on_map_management_page(hp: HomePage, lp: LoginPage):
     hp.to_map_management_page()
 
 
-@when(parsers.parse("I provides {name}, {privacy}, {description}, {latitude} and {longitude}"))
+@when(parsers.parse("I provide {name}, {privacy}, {description}, {latitude} and {longitude}"))
 def provide_map_details(mmp: MapManagementPage, mcp: MapCreatePage, name, privacy, description, latitude, longitude):
     mmp.to_map_create_page()
     mcp.create_a_map(name, description=description, latitude=latitude, longitude=longitude)
 
 
-@then(parsers.parse("I can successfully create a new map {name} without an error message"))
+@then(parsers.parse("I can successfully create {name} without an error message"))
 def create_map_successfully(mcp: MapCreatePage, name, mmp: MapManagementPage):
-    mcp.click_create()
```

Comment:
So this was basically the issue, because the button was already pressed in the function before, where I extracted the steps into one function create_a_map to be less repetetive but forgot to remove the click afterwards.

Reply:
Its really interesting though, how the tests initially passed with this logic.

Reply:
When we merged the branch the tests were passing and it was also with a double [click](https://github.com/ElektraInitiative/PermaplanT/blob/52381cc7514e1c7441bdc071808e51692376e77a/e2e/steps/test_map_creation.py).